servo-build-dependencies/arm.sls

```diff
@@ -36,7 +36,7 @@ arm-dependencies:
 libs-{{ target.name }}:
   archive.extracted:
     - name: {{ common.servo_home }}/rootfs-trusty-{{ target.name }}/{{ target.version }}
-    - source: https://servo-rust.s3.amazonaws.com/ARM/{{ target.download_name }}/{{ target.version }}/{{ target.download_name }}-{{ target.version }}.tgz
+    - source: https://servo-rust.s3.amazonaws.com/ARM/{{ target.version }}/{{ target.download_name }}.tgz
```

Comment:
Ugh, I'll try a third time later tonight.

On May 3, 2016, at 4:58 PM, Attila Dusnoki <notifications@github.com> wrote:

In servo-build-dependencies/arm.sls
<https://github.com/servo/saltfs/pull/347#discussion_r61963556>:

> @@ -36,7 +36,7 @@ arm-dependencies:
>  libs-{{ target.name }}:
>    archive.extracted:
>      - name: {{ common.servo_home }}/rootfs-trusty-{{ target.name }}/{{ target.version }}
> -    - source: https://servo-rust.s3.amazonaws.com/ARM/{{ target.download_name }}/{{ target.version }}/{{ target.download_name }}-{{ target.version }}.tgz
> +    - source: https://servo-rust.s3.amazonaws.com/ARM/{{ target.version }}/{{ target.download_name }}.tgz

if you want the original link, you still missing the
*-{{target.version}}.tgz from the end

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub
<https://github.com/servo/saltfs/pull/347/files/5303ab87b1e2bcb61ae42f3e27329ee677dd39d3#r61963556>